juriscraper/pacer/rss_feeds.py

```diff
@@ -314,6 +331,8 @@ def _get_docket_number(self, title_text):
                 self.docket_number_dist_regex,
                 self.docket_number_bankr_regex,
             ]
+        elif self.is_appellate:
+            regexes = [self.docket_number_bankr_regex]
```

Comment:
Let's either rename the variable or create a separate one for appellate, but let's not use the bankr. regex for appellate as is.